tools/exploredb/main.go

```diff
@@ -104,7 +104,7 @@ func main() {
 	case "migration-check":
 		destDbNameWithPath := filepath.Join(*destDatadir, *dbName)
 		if _, err := os.Stat(destDbNameWithPath); os.IsNotExist(err) {
-			log.Fatalf("could not locate destination database file : %s, %v", destDbNameWithPath, err)
+			log.WithError(err).WithField("path", dbNameWithPath).Fatal("could not locate database file")
```

Comment:
why the change here instead of using `destDbNameWithPath`?

Reply:
oh... copy paste error!